src/GovUk.Education.ExploreEducationStatistics.Admin/Migrations/ContentMigrations/20211111132702_EES2878UpdateReleaseTypeTitles.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Migrations.ContentMigrations
+{
+    public partial class EES2878UpdateReleaseTypeTitles : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.UpdateData(
+                table: "ReleaseTypes",
+                keyColumn: "Id",
+                keyValue: new Guid("1821abb8-68b0-431b-9770-0bea65d02ff0"),
+                column: "Title",
+                value: "Ad Hoc Statistics");
+
+            // To remove previous entry created by previous migration - EES-1500
+            migrationBuilder.DeleteData(
+                table: "ReleaseTypes",
+                keyColumn: "Id",
+                keyValue: new Guid("15bd4f57-c837-4821-b308-7f4169cd9330"));
+
+            // To remove previous entry created by previous migration - EES-1500
+            migrationBuilder.DeleteData(
```

Comment:
We won't be able to delete these if they are already in use. 

Would be safer to turn them into updates, assuming that the previous PR already inserted them.

```
            migrationBuilder.UpdateData(
                table: "ReleaseTypes",
                keyColumn: "Id",
                keyValue: new Guid("15bd4f57-c837-4821-b308-7f4169cd9330"),
                column: "Title",
                value: "Management Information");
```

If the migrations are baselined they should all appear in the baselined migration as Inserts.